kura/org.eclipse.kura.linux.systemd.provider/src/main/java/org/eclipse/kura/internal/linux/systemd/net/dns/LinuxDnsServerSystemD.java

```diff
@@ -12,11 +12,33 @@
  *******************************************************************************/
 package org.eclipse.kura.internal.linux.systemd.net.dns;
 
+import java.io.File;
+
+import org.eclipse.kura.KuraErrorCode;
+import org.eclipse.kura.KuraException;
 import org.eclipse.kura.internal.linux.net.dns.DnsServerService;
 import org.eclipse.kura.linux.net.dns.LinuxDnsServer;
 
 public class LinuxDnsServerSystemD extends LinuxDnsServer implements DnsServerService {
 
+    private static final String BIND9_COMMAND = "bind9";
+    private static final String NAMED_COMMAND = "named";
+    private static final String BIND9_SERVICE_UNIT_LOC = "/etc/systemd/system/bind9.service";
```

Comment:
It is the same as before, isn't it?

Reply:
pi@raspberrypi:~ $ ls -al /etc/systemd/system/bind9.service
ls: cannot access '/etc/systemd/system/bind9.service': No such file or directory

Reply:
I forgot the right path on your raspbian buster installation, can you recover it ?